elements_imaging/imaging.py

```diff
@@ -632,7 +734,7 @@ class ActivityExtractionMethod(dj.Lookup):
     extraction_method: varchar(32)
     """
 
-    contents = zip(['suite2p_deconvolution', 'caiman_deconvolution', 'caiman_dff'])
+    contents = zip(['suite2p_deconvolution', 'caiman_deconvolution', 'caiman_dff', 'miniscope_analysis'])
```

Comment:
perhaps you mean to add `miniscope_analysis_deconvolution` and `miniscope_analysis_dff` here?

Reply:
Thank you.  Fixed.